boards/arm/qemu_cortex_a53/qemu_cortex_a53_defconfig

```diff
@@ -0,0 +1,21 @@
+CONFIG_ARM=y
+CONFIG_ARM64=y
```

Comment:
Can we select this from the board/soc instead?

Reply:
ACK. Fix coming in V3. Closing to declutter the story. Thanks.